src/style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+}
+
+.header,
+.menu,
+.menu-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header {
+  box-sizing: border-box;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  background: url(./images/logo.png);
+  width: 40px;
+  height: 40px;
+}
+
+.nav-container,
+.menu,
+.menu-item,
+.ul-item {
+  padding: 0;
+  height: 100%;
+}
+
+.menu {
+  margin: 0;
+  list-style: none;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
```

Comment:
`line-height: 14px` can be removed from here

Reply:
> `line-height: 14px` can be removed from here

fixed